src/district_hegex/ui/core.cljs

```diff
@@ -105,3 +106,49 @@
     (-> (mount/with-args full-config)
       (mount/start))
     (re-frame/dispatch-sync [::init])))
+
+(defn  reinit []
+  (let [full-config (cljs-utils/merge-in
+                      config/config-map
+                      {:smart-contracts {:format :truffle-json}
+                       ;; :web3-account-balances {:for-contracts [:ETH :DNT]}
+                       :web3-tx-log {:tx-costs-currencies [:USD]
+                                     :default-settings {:from-active-address-only? true}}
+                       :reagent-render {:id "app"
+                                        :component-var #'router}
+                       :router {:routes routes
+                                :default-route :route/not-found
+                                :scroll-top? true}
+                       :notification {:default-show-duration 3000
+                                      :default-hide-duration 1000}})]
+
+    (println "dbg init0")
+
+    (js/console.log "config:" (clj->js full-config))
+    (-> (mount/with-args full-config)
+      (mount/start))
+    (re-frame/dispatch [::init])))
+
+
+(re-frame/reg-event-fx
+  ::listen-to-account-change
+  (fn []
```

Comment:
It appears like `::web3-accounts-events` lib listeners that detect account change are incompatible with new Metamask. I quickly checked out memefactory since it relies on the same listeners, it looks like it's not detecting account change there too (though address change is reflected in UI, I think this is because it uses a separate district0x module that outputs active account).
Hence this barebones-js listener workaround